modules/exploits/windows/local/one_day.rb

```diff
@@ -0,0 +1,314 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+require 'rex'
+require 'msf/core'
+
+# Removed some things that should be included...
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+  # errors with remove_socket
+  # include Msf::Exploit::Remote::SMB::Client::Psexec
+  include Rex::Constants::Windows
```

Comment:
This doesnt exist anymore, i believe it is now Msf::Exploit::Windows_Constants (right?).